packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -0,0 +1,275 @@
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-flipcard');
+const { generateTemplate } = storybookTemplate('sd-flipcard');
+const { overrideArgs } = storybookHelpers('sd-flipcard');
+
+export default {
+  title: 'Components/sd-flipcard',
+  component: 'sd-flipcard',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'front',
+      value: `<p slot="front">Lorem ipsum dolor sit amet per niente da faremmasds nonnummy dolore lorem ipsum dolor sit amet consectuer</p>`
+    },
+    {
+      type: 'slot',
+      name: 'back',
+      value: `<p slot="back">Lorem ipsum dolor sit amet per niente da faremmasds nonnummy dolore lorem ipsum dolor sit amet consectuer</p>`
+    },
+    {
+      type: 'slot',
+      name: 'mediaFront',
+      value: `<img slot='mediaFront' class="object-cover h-full w-full" src='./placeholders/images/generic.jpg' alt='Generic'/>`
+    },
+    {
+      type: 'slot',
+      name: 'mediaBack',
+      value: `<img slot='mediaBack' class="object-cover h-full w-full" src='./placeholders/images/generic.jpg' alt='Generic'/>`
+    }
+  ]),
+
+  argTypes,
+
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+/**
+ * This shows sd-flipcard in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`<div style="width:300px">${generateTemplate({ args })}</div>`;
+  }
+};
+
+/**
+ * The sd-flipcard can be displayed in several ways using the `frontVariant` and `backVariant` attributes. This example shows the usage `frontVariant` attribute.
+ */
+
+export const Variants = {
+  parameters: { controls: { exclude: ['frontVariant'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        y: {
+          type: 'attribute',
+          name: 'frontVariant'
+        }
+      },
+      args,
+      constants: [
+        {
+          type: 'template',
+          name: 'style',
+          value: '<div style="margin-bottom: 40px; width: 300px">%TEMPLATE%</div>'
+        }
+      ]
+    })
+};
+
+/**
+ * The sd-flipcard can be displayed in 3:4 or 16:9 aspect ratio.
+ */
+
+export const Ratio = {
+  parameters: { controls: { exclude: ['ratio'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'ratio'
+        }
+      },
+      args
+    }),
+  decorators: [
+    (story: any) =>
+      html`<style>
+          td.template {
+            width: 50%;
+          }
+        </style>
+        ${story()}`
+  ] as unknown
+};
+
+/**
+ * Use the `front`, `back`, `mediaFront` and `mediaBack` slots to add content to the flipcard.
+ */
+export const Slots = {
```

Comment:
for front and back, it shows a strange size of the slot. Probably correct but looks buggy.